.github/workflows/pytest.yml

```diff
@@ -0,0 +1,29 @@
+name: Run Python Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+env:
+  AIRFLOW_VERSION: 2.2.4
+  PYTHON_VERSION: 3.8.10
+  CONSTRAINT_URL: https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Python 3
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8.10
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt &&
+          pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"
```

Comment:
why these requirements are not in the `requirements.txt` file?

Reply:
Because I'm using official airflow image: https://github.com/ErnestaP/workflows-2/blob/workflow-376/Dockerfile#L1